core/src/test/scala/org/apache/spark/sql/delta/LogStoreSuite.scala

```diff
@@ -304,6 +304,15 @@ trait HDFSLogStoreSuiteBase extends LogStoreSuiteBase {
   protected def shouldUseRenameToWriteCheckpoint: Boolean = true
 }
 
+trait LocalLogStoreSuiteBase extends LogStoreSuiteBase {
```

Comment:
Is there a reason to have `LocalLogStoreSuiteBase`?

I think this would be cleaner if this code here just went into `class PublicLocalLogStoreSuite extends PublicLogStoreSuite { ...`

Reply:
> Is there a reason to have `LocalLogStoreSuiteBase`?
> 
> I think this would be cleaner if this code here just went into `class PublicLocalLogStoreSuite extends PublicLogStoreSuite { ...`

I am intending to extend this `LocalLogStoreSuiteBase` from `LocalLogStoreSuite` as well so that the code will be consistent with what we implemented on `PublicHDFSLogStoreSuite` and `HDFSLogStoreSuite`. 

The implementation of `LocalLogStoreSuite` would be changed like below, sorry for not updating it to this PR yet.
```java
class LocalLogStoreSuite extends LocalLogStoreSuiteBase {
  override val logStoreClassName: String = classOf[LocalLogStore].getName
}
```